as_perimenoun_oi_gynaikes.ass

```diff
@@ -995,23 +996,23 @@ Dialogue: 0,1:11:51.01,1:11:53.44,Default,,0,0,0,,{\i1}It is my fault.{\i0}
 Dialogue: 0,1:11:53.99,1:11:59.36,Default,,0,0,0,,{\i1}Why shouldn't I stop you from leaving?{\i0}
 Dialogue: 0,1:11:59.36,1:12:03.69,Default,,0,0,0,,{\i1}Stand at the door and not let you go.{\i0}
 Dialogue: 0,1:12:05.26,1:12:10.84,Default,,0,0,0,,{\i1}Why shouldn't I stop you from leaving?{\i0}
-Dialogue: 0,1:12:10.84,1:12:16.74,Default,,0,0,0,,{\i1}Why shouldn't I call 911 and tell dispatch...{\i0}
+Dialogue: 0,1:12:10.84,1:12:16.74,Default,,0,0,0,,{\i1}Why shouldn't I call the police and tell to the officer...{\i0}

```

Comment:
We are not going for 100% accuracy but for something that would make sense for the international reader and relates to the movie. Call 100 in Greece would be call 911 in an international context. When you call 911 you get dispatch, not an officer.